### PR TITLE
add PyXB

### PR DESCRIPTION
**Pull request by grlee77**

### CODE REVIEW

**Comment by grlee77** [comment]
don't know why the import test is failing on Windows.  this is a pure python recipe

File: recipes/pyxb/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+  run:
+    - python
+
+test:
+  requires:
+    - nose
+  imports:
+    - pyxb
+  commands:
+    - cd ${SRC_DIR}/tests && nosetests *
```
Comment (synapticarbors) [inline recipes/pyxb/meta.yaml]: I think you need to use different `cd` syntax for windows and non-windows. See the following for an example:

https://github.com/conda-forge/staged-recipes/pull/1138/files#diff-359a956b2fb22cc654bc16d8e80cff8aR30

File: recipes/pyxb/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+
+about:
+  home: http://pyxb.sourceforge.net
+  license: Apache 2.0
+  summary: PyXB ("pixbee") is a pure Python package that generates Python source code for classes that correspond to data structures defined by XMLSchema.
```
Comment (synapticarbors) [inline recipes/pyxb/meta.yaml]: Please add:

- `dev_url: https://github.com/pabigot/pyxb`
- `license_file: LICENSE`

**Comment by grlee77** [comment]
The test suite will not pass on Windows due to https://github.com/pabigot/pyxb/issues/34, but there is a workaround to the issue, so I think we should go ahead and leave the windows build enabled.

File: recipes/pyxb/meta.yaml (hunk 2 above)
Comment (jakirkham) [inline recipes/pyxb/meta.yaml]: Done below.

**Comment by jakirkham** [comment]
Thanks @grlee77 .

**Pull request merged**